### ISSUE TITLE
Expand touch sensible area

### ISSUE THREAD

**Issue by lianergoist** [tuskyapp/Tusky#3033]
In both horizontal and vertical mode, there is an area on both sides of the message area, where there is no touch sensiblility (the dark space). It would be nice if we could use that area to scroll. 

![image_2022_12_10_07_02_07](https://user-images.githubusercontent.com/6023573/206836537-3c35b012-ca44-4c2f-ad71-ca7cbe32ee4d.jpg)
![image_2022_12_10_06_58_41](https://user-images.githubusercontent.com/6023573/206836538-5252d3df-0886-4a3f-9d6d-6d1ed1eb7a6a.jpg)

* * * *
- Tusky Version: 20.0
- Android Version: 12
- Android Device: Lenovo 2000*1200

**Comment by GPHemsley** [comment]
Indeed, I have found myself accidentally hitting it instead of the button I'm trying to hit, thereby navigating to the post instead.

**Comment by beholdnec** [comment]
When I scroll any app on my device, I naturally want to hold the device in one hand and use my thumb to swipe near the edge of the screen.

The fact that Tusky has uninteractable columns on the left and right edges of the screen means I have to reach farther than usual to scroll and risk accidentally tapping interactable elements. Hoping to see this addressed in the near future.
